varcode/transcript_mutation_effects.py

```diff
@@ -313,7 +315,7 @@ def get_class(effect):
             return _MultipleSubstitution
     return effect.__class__
 
-variant_effect_priority_list = [
+transcript_effect_priority_list = [
```

Comment:
I agree this is hacky, I wasn't sure how to do it better. I'm a little confused about your first suggestion, how is it different from the `top_priority_transcript_effect` function below the priority list?

As for the second suggestion, that's actually how I started out. However, every time I added a variant I had to go looking around every existing class to figure out what priority number to assign the new class, sometimes having to shift around existing priorities. Seems strictly less maintainable/understandable than having a list of classes.